### DISCUSSION TITLE
Hide one-time code input most of the time

### DISCUSSION THREAD

**Discussion by alpha-pix** [snicco/fortress#8] [Feature Requests]
One time code input is clutter 99.99% of the time. Please hide this option on a separate page behind a link, like lost password.

**Comment by calvinalkan** [comment]
I assume you mean "hide the recovery code field" on the 2FA Page?

**Reply by alpha-pix** [reply]
Yes!

**Comment by calvinalkan** [comment]
This won't be handled until we do a major UI redesign with HTMX/ajax.
